<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsWindow</class>
 <widget class="QWidget" name="OptionsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {background-image: url(gfx/button1.png);}
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>234</y>
     <width>250</width>
     <height>300</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>250</horstretch>
     <verstretch>29</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>300</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>250</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane { margin: 0px,1px,1px,1px; border: 1px solid black; border-radius: 7px; padding: 1px;}
QWidget{background-color: transparent;}</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <widget class="QWidget" name="gameTab">
    <attribute name="title">
     <string>   Game   </string>
    </attribute>
   </widget>
   <widget class="QWidget" name="soundTab">
    <attribute name="title">
     <string>  Sound   </string>
    </attribute>
   </widget>
   <widget class="QWidget" name="graphicsTab">
    <attribute name="title">
     <string> Graphics</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>710</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>710</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
